### ISSUE TITLE
Making vscode.executeCommand('vscode.previewHtml', ...) a compile error

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/vscode#62806]
Part of #62630 

Discourage people from writing the most simple form of the `vscode.previewHtml` command by making:

```ts
vscode.executeCommand('vscode.previewHtml')
```

A compile error. Obviously this compile time check won't block sneaky code like:

```ts
const command = 'vscode.previewHtml'
vscode.executeCommand(command)
```

or:

```ts
vscode.executeCommand('vscode' + '.' + 'previewHtml')
```

We've already added a runtime warning for development extensions that will catch the above examples and will upgrade this into a user facing warning after a few months

**Comment by mjbvz** [comment]
Yeah this does not work well using an overload since most cases aren't caught. We have to revert the proper fix, 96ddd4f288c627550f4d467f14dc556785c5d962, because the conditional types broke our build (and would have broken anyone using TS 2.8 or older to target newer VS Code versions).

I'm just going to revert the overload too and mark this as not worth the effort

**Comment by isidorn** [comment]
@mjbvz ok. Removing verification-needed than
